### PR TITLE
test pr for CLA issues

### PR DESCRIPTION
**Pull request by Hoten**
I wanna fix this.

### CODE REVIEW

File: lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js [hunk 1]
```diff
@@ -439,6 +439,7 @@ class NetworkAnalyzer {
    */
   static findMainDocument(records, finalURL) {
     // Try to find an exact match with the final URL first if we have one
+    // .....
```
Comment (Hoten) [inline lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js]:
```suggestion
    // ..... ...
```

File: lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js [hunk 2]
```diff
@@ -439,6 +439,7 @@ class NetworkAnalyzer {
    */
   static findMainDocument(records, finalURL) {
     // Try to find an exact match with the final URL first if we have one
+    // ..... ...
```
Comment (patrickhulce) [inline lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js]:
```suggestion
    // go *warriors*
```

**Comment by Hoten** [comment]
image

@patrickhulce is this email of yours in https://cla.developers.google.com/clas ?

**Comment by Hoten** [comment]
The CLA checking code added supported for Co-Authored 2-3 months ago, so that should be fine these days.

**Comment by brendankenny** [comment]
> @patrickhulce is this email of yours in https://cla.developers.google.com/clas ?

yes

The "Googlers: Go here" link has links to where you can check that

**Comment by patrickhulce** [comment]
@Hoten yessir that's the one!

image

**Comment by Hoten** [comment]
b/134532885

**Pull request closed without merging**